artman/tasks/publish/local.py

```diff
@@ -73,7 +73,7 @@ def execute(self, gapic_code_dir, git_repo, local_paths,
         dests = []
 
         # Determine where the code belongs and stage it there.
-        for path in git_repo.get('paths', ['.']):
```

Comment:
Changed to raise an error. I am not sure what spots need to be changed to add the `.` path - I can't see any usage that relies on it (for cases such as DLP where it is currently missing, we want to fail).